src/Beberlei/Metrics/Collector/Collector.php

```diff
@@ -18,9 +18,38 @@
  */
 interface Collector
 {
+    /**
+     * Update one or more counters by some arbitrary amount
```

Comment:
``Updates``

And it updates a single counter, not several ones, as far as I can tell